config/default/kustomization.yaml

```diff
@@ -39,104 +49,104 @@ patches:
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'.
 # Uncomment 'CERTMANAGER' sections in crd/kustomization.yaml to enable the CA injection in the admission webhooks.
 # 'CERTMANAGER' needs to be enabled to use ca injection
-- path: webhookcainjection_patch.yaml
+# - path: webhookcainjection_patch.yaml
 
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER' prefix.
 # Uncomment the following replacements to add the cert-manager CA injection annotations
-replacements:
```

Comment:
What implication does it have to disable those replacements? Will all existing deployments still work? 🤔

Reply:
Maybe it might affect other deployments, so I reverted them.